tests/performance/aggregating_merge_tree.xml

```diff
@@ -0,0 +1,32 @@
+<test>
+    <create_query>DROP TABLE IF EXISTS test</create_query>
```

Comment:
Perf test script runs the drop queries before create queries, so you don't have to do this manually.